api/types/types.proto

```diff
@@ -2609,6 +2609,39 @@ message GithubConnectorSpecV3 {
     string Display = 5 [ (gogoproto.jsontag) = "display" ];
 }
 
+// SSOInfoType represents the SSO diagnostic info type.
+enum SSOInfoType {
+    // Unknown info type.
+    UNKNOWN = 0;
+    // Error info type, used for reporting errors.
+    ERROR = 1;
+    // Success info type, used for marking successful flows.
+    SUCCESS = 2;
+    // Create user params, contains info about attempted user creation parameters.
+    CREATE_USER_PARAMS = 3;
+    // [SAML] Attributes to roles mapping.
+    SAML_ATTRIBUTES_TO_ROLES = 4;
```

Comment:
Do we want add some descriptor number space between common info types and known connector types ? so in case where a new type needed by added the values will be still aggregated.

Reply:
Sure, why not. Done: https://github.com/gravitational/teleport/pull/11508/commits/19c4c326dfb73dee2cf4adef94af1b11728c612f